docs/source/release_notes.rst

```diff
@@ -2,6 +2,8 @@ Release Notes
 -------------
 **Future Releases**
     * Enhancements
+        * Added oversamplers to AutoMLSearch :pr:`2213`
+        * Added tests for model understanding for oversamplers :pr:`2286`
```

Comment:
Mega nit-pick: you could lump these into one line if you want and our release notes job will still pass:
```
* Added oversamplers to AutoMLSearch :pr:`2213` :pr:`2286`
```